action.yml

```diff
@@ -107,42 +107,42 @@ runs:
       shell: bash
       run: |
         cat <<EOF >> $GITHUB_ENV
-          RELEASE_POST_SCRIPTS="${{ inputs.mavenPostScripts }}"
-          RELEASE_PRE_SCRIPTS="${{ inputs.mavenPreScripts }}"
-          MAVEN_BIN="${{ inputs.mavenBinary }}"
-          MAVEN_PROJECT_POM="${{ inputs.mavenProjectPom }}"
-          RELEASE_BRANCH_NAME="${{ inputs.releaseBranchName }}"
-          MAVEN_RELEASE_SETTINGS_XML="${{ inputs.mavenReleaseSettingsXml }}"
-          MAVEN_RELEASE_SERVER_ID="${{ inputs.mavenReleaseServerId }}"
-          MAVEN_RELEASE_SERVER_USERNAME="${{ inputs.mavenReleaseServerUsername }}"
-          MAVEN_RELEASE_SERVER_PASSWORD="${{ inputs.mavenReleaseServerPassword }}"
-          MAVEN_SNAPSHOT_SERVER_ID="${{ inputs.mavenSnapshotServerId }}"
-          MAVEN_SNAPSHOT_SERVER_USERNAME="${{ inputs.mavenSnapshotServerUsername }}"
-          MAVEN_SNAPSHOT_SERVER_PASSWORD="${{ inputs.mavenSnapshotServerPassword }}"
-          MAVEN_RELEASE_EXECUTE_PERFORM="${{ inputs.mavenReleaseExecutePerform }}"
-          MAVEN_RELEASE_PUSH_COMMITS="${{ inputs.mavenReleasePushCommits }}"
-          MAVEN_RELEASE_EXECUTE_POST_SCRIPTS="${{ inputs.mavenReleaseExecutePostScripts }}"
-          MAVEN_RELEASE_EXECUTE_PRE_SCRIPTS="${{ inputs.mavenReleaseExecutePreScripts }}"
-          MAVEN_RELEASE_ARGUMENTS="${{ inputs.mavenReleaseArguments }}"
-          GIT_USER_NAME="${{ inputs.gitUserName }}"
-          GIT_USER_EMAIL="${{ inputs.gitUserEmail }}"
-          MAVEN_RELEASE_TAG="${{ inputs.mavenReleaseTag }}"
-          MAVEN_RELEASE_VERSION="${{ inputs.mavenReleaseVersion }}"
-          MAVEN_NEXT_DEVELOPMENT_VERSION="${{ inputs.mavenNextDevelopmentVersion }}"
+        RELEASE_POST_SCRIPTS=${{ inputs.mavenPostScripts }}
```

Comment:
why removing quotes? Does `${{` take care of it? Or is it yaml syntax? (sorry for being noob :) )

Reply:
heredoc preserve quotes (`echo key="${value}"` doesn't) and they were printed to file which was incorrect. no quotes needed there

Reply:
so the variable (like `inputs.mavenPreScripts`) has the quotes already in the value?